src/tests/snapshotTests/calculator.test.js

```diff
@@ -0,0 +1,10 @@
+import renderer from 'react-test-renderer';
+// import calculator from '../../Components/calculator';
+
```

Comment:
- [ ] You seem to have added the **React Testing Library** package in your **`package.json`** file. However, you are still missing tests using this library. Kindly add tests for simulating user interaction in your app using RTL.

  You can refer to these links, [RTL Intro](https://testing-library.com/docs/react-testing-library/intro/) and [RTL Tutorial](https://www.robinwieruch.de/react-testing-library/), for ideas on how to do this.